skbio/stats/composition.py

```diff
@@ -1288,10 +1514,10 @@ def sbp_basis(sbp):
     ...                 [0, 0, 0, 1,-1]])
     ...
     >>> sbp_basis(sbp)
-    array([[ 0.31209907,  0.31209907,  0.12526729,  0.12526729,  0.12526729],
-           [ 0.36733337,  0.08930489,  0.18112058,  0.18112058,  0.18112058],
-           [ 0.17882092,  0.17882092,  0.40459293,  0.11888261,  0.11888261],
-           [ 0.18112058,  0.18112058,  0.18112058,  0.36733337,  0.08930489]])
+    array([[ 0.54772256,  0.54772256, -0.36514837, -0.36514837, -0.36514837],
```

Comment:
This looks like a pretty substantial change in results?

Reply:
Correct, this is a change from probabilities to log-probabilities, which is better for numerical stability.

Reply:
Thanks. Can this behavior change be explicitly noted in the changelog?